metricflow-semantics/metricflow_semantics/experimental/semantic_graph/builder/measure_subgraph.py

```diff
@@ -25,21 +25,21 @@
 logger = logging.getLogger(__name__)
 
 
-class MeasureSubgraphGenerator(SemanticSubgraphGenerator):
-    """Generate the subgraph that relates measures to other entities.
+class SimpleMetricSubgraphGenerator(SemanticSubgraphGenerator):
+    """Generate the subgraph that relates simple metrics to other entities.
 
-    For each measure, this adds the following edges:
+    For each simple metric, this adds the following edges:
 
-    * MeasureNode -> LocalModelNode (corresponding to the semantic model where the measure is defined)
-    * MeasureNode -> MetricTimeNode (edge describes aggregation time grain)
+    * SimpleMetricNode -> LocalModelNode (corresponding to the semantic model where the simple metric is defined)
+    * SimpleMetricNode -> MetricTimeNode (edge describes aggregation time grain)
     """
 
     @override
     def add_edges_for_manifest(self, edge_list: list[SemanticGraphEdge]) -> None:
         for measure_lookup in self._manifest_object_lookup.simple_metric_model_lookups:
```

Comment:
nit: still uses `measure_lookup` as the variable name (might be fixed up the stack)